docs/configuration.rst

```diff
@@ -365,32 +421,45 @@ with the next two options:
       * Jungle wood and jungle leaves have id 17 and 18 and use data value 3 for first two bits (bitmask 3 = 0b11)
       * other bits are used otherwise -> ignore all those bits
 
+-----
 
 Map Options
 -----------
 
 .. note::
 
-    These options are for the maps. You can specify them in the map sections
-    (the ones starting with map:) or you can specify them in the global:map
-    section.  If you specify them in the global section, these options are
-    default values and inherited into the map sections if you do not overwrite
+    These options are for maps. You can specify them in the map sections
+    (the ones starting with ``map:``) or you can specify them in the ``global:map``
+    section.  If you specify them in the global section, these options become
+    default values and are inherited into the map sections if you do not overwrite
     them.
 
-``name = <name>``
+**Name:** ``name = <name>``
 
     **Default:** ``<name of the section>``
 
-    This is the name for the rendered map. You will see this name in the output
-    file, so you should use here an human-readable name. The belonging
-    configuration section to this map has also a name (in square brackets).
-    Since the name of the section is used for internal representation, the name
-    of the section should be unique and you should only use alphanumeric chars.
+    .. image:: img/map_name.png
+       :align: center
+       :alt: Your map name appears in the drop down list of maps.
 
-``render_view = isometric|topdown``
+    This is the name for the rendered map. You will see this name in the dropdown 
+    list of maps, so you should use a human-readable name (spaces, numbers, symbols, 
+    even unicode are all OK). 
+    
+    The configuration section for this map has also a name (in square brackets).
+    This section name is used for internal representation, command line arguments, 
+    and on-disk directory names, so the section name should be unique and only use
+    alphanumeric chars and underscore (definitely no spaces).
+
+
+**Render View:** ``render_view = isometric|topdown|side``
 
     **Default:** ``isometric``
 
+    .. image:: img/map_render_view.png
```

Comment:
I think I have an idea for this: What about a table with different render views in left column and images in right column? That way description and examples are close together. Same for render modes and overlays.

Reply:
I knew I'd be capturing all the screenshots again (even just for `daylight` instead of `plain`). One of the rules of documentation: screenshots get re-done 3 times!! I'll put tables together and we can see how they look - probably a bit more vertical space, but better quality images.